lms/templates/navigation-edx.html

```diff
@@ -142,7 +142,9 @@ <h1 class="logo" itemscope="" itemtype="http://schema.org/Organization">
 <![endif]-->
 % endif
 
-%if not user.is_authenticated():
+## The forgot password modal JavaScript assumes that JQuery is loaded,
+## which is not necessarily the case when using RequireJS.
+%if not enable_require_js and not user.is_authenticated():
   <%include file="forgot_password_modal.html" />
```

Comment:
But if enable_require_js is true, then the forgot_password_modal.html file is not included. So how does it work in that case?

And I'm confused-- how is enable_require_js set? Will it only be true on specific pages?